### DISCUSSION TITLE
Welcome to F_2109 Discussions!

### DISCUSSION THREAD

**Discussion by namikis** [jphacks/F_2109#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by namikis** [comment]
てすと

**Reply by kenkenpa2126** [reply]
リプライ

**Comment by kenkenpa2126** [comment]
テスト

**Reply by namikis** [reply]
@kenkenpa2126 
test
